src/main/resources/application.yml

```diff
@@ -1,3 +1,7 @@
+server:
+  servlet:
+    context-path: /planner
```

Comment:
Property to define application context path.
For example: 
`http://localhost:8080/planner/vacation`
`http://localhost:8080/planner/students/123`